src/org/opensolaris/opengrok/history/HistoryGuru.java

```diff
@@ -73,6 +73,8 @@
 
     private final int scanningDepth;
 
+    private String ignoreRepositoryName = ".opengrok_skip_history";
```

Comment:
Vlad, I am at a total loss right now. I am looking and answering this message via github web site and i don't see what pull request you are mentioning. (Or even how to go about looking at it)